components/salsa-macros/src/query_group.rs

```diff
@@ -681,13 +681,13 @@ impl TryFrom<syn::Attribute> for SalsaAttr {
     type Error = syn::Attribute;
 
     fn try_from(attr: syn::Attribute) -> Result<SalsaAttr, syn::Attribute> {
-        if is_not_salsa_attr_path(&attr.path) {
+        if is_not_salsa_attr_path(&attr.path()) {
             return Err(attr);
         }
 
         let span = attr.span();
-        let name = attr.path.segments[1].ident.to_string();
-        let tts = attr.tokens.into();
+        let name = attr.path().segments[1].ident.to_string();
+        let tts = attr.into_token_stream().into();
```

Comment:
Doesn't this return the entire attribute? Previously `tokens` were the tokens that are not the path e.g. `(foo)` in `salsa::something(foo)`.

Reply:
Yeah, that's probably it. I looked at `attr.meta`, but couldn't really make it work.

Reply:
The `MetaList` works differently. It doesn't have the path, but it doesn't have the parenthesis either. So you would have `foo`. You could probably `quote!` this and wrap the result around parentheses as a dirty hack.

Reply:
This is actually working now AFAICT, it fails to build because of some `Span`/`DelimSpan` changes.